snabbdom/src/main/scala/snabbdom/VNode.scala

```diff
@@ -62,13 +62,17 @@ class VNode private (
 
 object VNode {
 
+  val empty =
+    new VNode(None, VNodeData.empty, None, None, Some(""), None, None)
```

Comment:
Isn't this global mutable state?

Reply:
Oops, should be a `def`

Reply:
I wonder if we should make it `empty()` to distinguish it as allocating mutable state 🤔